applications/openshift/api-server/api_server_audit_log_maxsize/kubernetes/shared.yml

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_ocp
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  name: cluster
+spec:
+  maximumFileSizeMegabytes: 100
```

Comment:
nice! This looks really good! To ensure that this gets tested and we don't regress, do you mind adding an e2e test [1]?

The main idea is to check what was the default value and to check that the check should pass once it gets remediated.

So...

if the test was failing to begin with and then it gets remediated, the `e2e.yml` file would look as follows:

```
---
default_result: FAIL
result_after_remediation: PASS
```

There is no need to create an `e2e-remediation.sh` file since this already has an automated remediation, which is this file.

[1] https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#end-to-end-tests

Reply:
Should `maximumFileSizeMegabytes` be variable-ized and default to 100? (Not sure if we can do that substitution with remediations).

Reply:
We currently can't. We would need an enhancement in the OpenSCAP command to be able to do those. The problem is that variables only get replaced on the remediate command from `oscap` and since these are `kubernetes` remediations we would need to enhance how that works... then hook that up to the operator. Certainly something we should tackle at some point.

Reply:
@JAORMX Will get this `e2e.yml` test in.

Reply:
@swarred Seems that there was already an e2e.yml and the test was passing to begin with. This addition actually makes the test fail. Let's remove the changes to e2e.yml and keep the test passing, as it seems that the default was just fine.